packages/create-redwood-app/package.json

```diff
@@ -14,8 +15,7 @@
   ],
   "scripts": {
     "build": "yarn node ./build.mjs",
-    "build:watch": "nodemon --watch src --ignore dist,template --exec \"yarn build\"",
-    "prepublishOnly": "NODE_ENV=production yarn build"
+    "test": "yarn node --test"
```

Comment:
esbuild sets `NODE_ENV` to production if minify is true, and here it is. See https://esbuild.github.io/api/#platform. (But more generally, I'm  sure `NODE_ENV` really does anything here.)